client/state/utils/package.json

```diff
@@ -0,0 +1,3 @@
+{
+	"sideEffects": false
```

Comment:
😮 Is this enough to trigger webpack tree shaking for some directory tree?

Reply:
It is indeed! Anything under that point will be tree-shaken. It blew my mind when @jsnajdr told me about it too 🤯